holonote/annotate/table.py

```diff
@@ -20,8 +20,6 @@ class AnnotationTable(param.Parameterized):
 
     columns = ("region", "dim", "value", "_id")
 
-    index = param.List(default=[])
```

Comment:
This class is not used as a parameterized class and removes the method `_update_index` for a property that looks up the index.